ports/icu/portfile.cmake

```
@@ -406,16 +411,17 @@ file(INSTALL ${ICU_TOOLS} DESTINATION ${CURRENT_PACKAGES_DIR}/tools/${PORT})
 
 # To cross compile, we need some files at specific positions. So lets copy them
 file(GLOB CROSS_COMPILE_TOOLS ${CURRENT_PACKAGES_DIR}/lib/icu*${ICU_VERSION_MAJOR}.dll)
-list(APPEND CROSS_COMPILE_TOOLS 
-${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/derb${VCPKG_HOST_EXECUTABLE_SUFFIX}
-${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/genbrk${VCPKG_HOST_EXECUTABLE_SUFFIX}
-${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/gencfu${VCPKG_HOST_EXECUTABLE_SUFFIX}
-${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/gencnval${VCPKG_HOST_EXECUTABLE_SUFFIX}
-${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/gendict${VCPKG_HOST_EXECUTABLE_SUFFIX}
-${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/genrb${VCPKG_HOST_EXECUTABLE_SUFFIX}
-${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/makeconv${VCPKG_HOST_EXECUTABLE_SUFFIX}
-${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/icupkg${VCPKG_HOST_EXECUTABLE_SUFFIX}
-${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/pkgdata${VCPKG_HOST_EXECUTABLE_SUFFIX})
+list(APPEND CROSS_COMPILE_TOOLS
+    ${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/derb${VCPKG_HOST_EXECUTABLE_SUFFIX}
+    ${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/genbrk${VCPKG_HOST_EXECUTABLE_SUFFIX}
+    ${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/gencfu${VCPKG_HOST_EXECUTABLE_SUFFIX}
+    ${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/gencnval${VCPKG_HOST_EXECUTABLE_SUFFIX}
+    ${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/gendict${VCPKG_HOST_EXECUTABLE_SUFFIX}
+    ${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/genrb${VCPKG_HOST_EXECUTABLE_SUFFIX}
+    ${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/makeconv${VCPKG_HOST_EXECUTABLE_SUFFIX}
+    ${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/icupkg${VCPKG_HOST_EXECUTABLE_SUFFIX}
+    ${CURRENT_BUILDTREES_DIR}/${RELEASE_TRIPLET}/bin/pkgdata${VCPKG_HOST_EXECUTABLE_SUFFIX}
+)
 file(COPY ${CROSS_COMPILE_TOOLS} DESTINATION ${CURRENT_PACKAGES_DIR}/tools/${PORT}/for-cross-compile/bin)
```

```suggestion
file(COPY ${CROSS_COMPILE_TOOLS} DESTINATION ${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin)
```
i think the additional subfolder is not needed. Or is there a reason to have it?

---

The only requirement is, that there is a folder `bin` with the executables and next to `bin` folder a `config`folder with the icucross.* files. I have chosen an extra `for-cross-compile` folder to make the intention of the folder content clear, otherwise it might be a little bit strange why the executables are duplicated in a bin folder next to the executables.